app/producers/application_producer.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Common Kafka producer client
+# https://www.rubydoc.info/gems/ruby-kafka/Kafka/Producer
+class ApplicationProducer < Kafka::Client
+  BROKERS = [ENV['KAFKAMQ']].compact.freeze
+  CLIENT_ID = 'compliance-payload-tracker-producer'
```

Comment:
@akofink Can the `CLIENT_ID` be set to something else, like "compliance-backend"?

Reply:
Changed to common "compliance-backend". Defining per producer isn't necessary, as there are different topics, unless client instances would collide.